TruthTables/Tasks.qs

```diff
@@ -44,19 +44,23 @@ namespace Quantum.Kata.TruthTables {
 
     // Task 1. Projective functions (elementary variables)
     //
-    // Goal: Describe the three projective functions x₁, x₂, x₃ as 3-input
-    //       functions, represented by integers.  Note that we follow the
-    //       convention that x₁ is the least-significant input.
+    // Goal: Describe the three projective functions f₁, f₂, f₃ represented
+    // by integers. They are 3 input functions i.e. f(x) : {0,1}ⁿ → {0,1},
+    // n being equal to 3.
     //
-    // Example: The function x₁ (least-significant input) is given as an
-    //          example.  The function is true for assignments 001, 011, 101,
-    //          and 111.
+    // f₁ should be true for the following assignments : 001, 011, 101 and 111.
```

Comment:
I think it's a lot less interesting to list the inputs for which the function should be true than to explain what this function is (function f₁ returns the first input, f₂ - the second one, etc.) and let the learner figure out this list themselves. That's why we give one example but not all three.

I would make it more clear what x₂ and x₃ are, but not spell out for which inputs they should be 1.

Reply:
Agree. I didn't read the comments in order and gave a similar comment below.